tests/test_transforms/test_timestamp/test_holiday_transform.py

```diff
@@ -41,6 +41,21 @@ def two_segments_simple_ts_daily(simple_constant_df_daily: pd.DataFrame):
     return ts
 
 
+@pytest.fixture()
+def two_segments_simple_ts_day_15min(simple_constant_df_daily: pd.DataFrame):
```

Comment:
This function returns bad `TSDataset`, because you create pd.DataFrames with `freq="D"` and in `TSDataset` you define `freq="1D 15MIN"`.